config/kinto.ini

```diff
@@ -0,0 +1,149 @@
+# Created at Tue, 15 Mar 2016 11:24:21 +0100
+# Using Kinto version 2.1.0.dev0
+
+
+[server:main]
+use = egg:waitress#main
+host = 0.0.0.0
+port = 8888
+
+
+[app:main]
+use = egg:kinto
+
+#
+# Backends.
+#
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#storage
+#
+kinto.storage_backend = cliquet.storage.memory
+kinto.storage_url =
+kinto.cache_backend = cliquet.cache.memory
+kinto.cache_url =
+kinto.permission_backend = cliquet.permission.memory
+kinto.permission_url =
+
+#
+# Auth configuration.
+#
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#authentication
+#
+kinto.userid_hmac_secret = d32712c82034ec704828ff9038c5926fca11463fb92130dd70d8ae7d593294c5
```

Comment:
You mean in [kinto.tpl](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl)?